script.js

```diff
@@ -0,0 +1,28 @@
+let blurredAreaModal = document.querySelector('.modal-blurred-area')
+let showInstructionsButton = document.querySelector('#instruction-button')
+let instructionsModal = document.querySelector('.instruction-modal')
+let instructionsBody = document.querySelector('.instructions-body')
+let hideInstructionsModal = document.querySelector('#modal-button')
+
+showInstructionsButton.addEventListener('click', (event) => {
+    event.preventDefault()
```

Comment:
You _probably_ don't need to have preventDefault and stopPropagation here. I would think just stopProp will do the trick